webapp/store/user/hooks.js

```diff
@@ -25,6 +25,9 @@ export const useAuthCanViewTemplates = () => Authorizer.canViewTemplates(useUser
 export const useAuthCanCreateTemplate = () => Authorizer.canCreateTemplate(useUser())
 export const useAuthCanEditTemplates = () => Authorizer.canEditTemplates(useUser())
 
+// ====== Auth / Analysis
+export const useAuthcanUseAnalysis = () => Authorizer.canAnalyzeRecords(useUser(), useSurveyInfo())
```

Comment:
very stupid thing, useAuthcanUseAnalysis should be useAuthCanUseAnalysis, just to be consistent with the other hooks